api/Node.json

```diff
@@ -543,10 +543,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This is `getFeature`, and the only null&rarr;false change in this PR. I trust that it's right, and we should remove this entry since it's not implemented anywhere.